app/javascript/src/lib/translations.ts

```diff
@@ -0,0 +1 @@
+export const createNewTranslationKey = Symbol("create new translations key")
```

Comment:
Why did you choose a symbol here?

Reply:
A nullish `selectedKey` means "no key selected". A `selectedKey` that is a string means the key of that name if selected. There were no other (sensible) values left, which is where Symbols come in (I think).